run-build.sh

```diff
@@ -93,6 +96,14 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --linux-portable)
+            # Pass in a specific version (the one we set up to bootstrap this experience). Once 
+            # we have an official build we can remove the version parameter and use the latest.
+            LINUX_PORTABLE_INSTALL_ARGS="--linux-portable --version 2.0.0-alpha-005151"
```

Comment:
IMO, it is better to just pin the stage0 to a specific build for all platforms (windows and unix).  That way our build is consistent across everything.

See https://github.com/dotnet/cli/pull/5839 (the non-README changes) for what unpinning stage0 looks like.